pkg/broker/broker.go

```diff
@@ -366,13 +409,8 @@ func (a AnsibleBroker) Bind(instanceUUID uuid.UUID, bindingUUID uuid.UUID, req *
 	//
 	// See if the service instance still exists, if not send back a badrequest.
 
-	instance, err := a.dao.GetServiceInstance(instanceUUID.String())
+	instance, err := a.getServiceInstance(instanceUUID)
```

Comment:
```a.getServiceInstance``` takes a string as input.  I think you still need ```instanceUUID.String()```. https://github.com/openshift/ansible-service-broker/blob/master/pkg/dao/dao.go#L190